src/samples/AppSelector/index.tsx

```diff
@@ -1,7 +1,6 @@
 import React from "react";
 import { Switch, Route } from 'react-router-dom';
-import EmbeddedTopLevel from "../Embedded/EmbeddedTopLevel";
-import FullPortal from "../FullPortal";
+import ChildBenefits from '../ChildBenefits'
```

Comment:
Can we name the sample 'ChildBenefitClaim' to align with the case type name in Pega Platform and use this pattern for any future samples which correspond to case types in Pega